### ISSUE TITLE
WriteBlobWriteParams could not be found (even doe the js was loaded)

### ISSUE THREAD

**Issue by J4L0** [KristofferStrube/Blazor.FileSystem#4]
I was trying to adjust the WriteAsync functions a little (adding a write for a DotNetStreamReference) and stumbled over the fact, that my method added to the js file wasn't found. Same goes for the existing version using "WriteBlobWriteParams".

Am I doing something wrong? Was it tested, cause there is no example using the above. 
Could it reference a different IJSObjectReference?

The file is loaded as of what I see in the Devtools sources tab, but I receive an error msg:

Microsoft.JSInterop.JSException: Could not find 'WriteBlobWriteParams' ('WriteBlobWriteParams' was undefined).
blazor.webassembly.js:1 Error: Could not find 'WriteBlobWriteParams' ('WriteBlobWriteParams' was undefined).

Thanks for your reply

**Comment by KristofferStrube** [comment]
Hey @J4L0, you are right. When `FileSystemWritableFileStream` was changed to inherit from `Blazor.Streams`'s `WritableStream` class instead of just `Stream` in [version 2.0.0 of Blazor.FileSystemAccess](https://github.com/KristofferStrube/Blazor.FileSystemAccess/blob/main/CHANGELOG.md#200---2022-11-10) I have accidentally also used the inherited helper JS file which originates from `Blazor.Streams`. But I have also gotten smarter since I wrote this originally a year ago and now have a way to parse the `Blob` argument from `BlobWriteParams` to the `write` method without a dedicated JS method. I will get that out very soon which should fix your immediate problems.

Apart from this I hear that you are looking into trying to write a `DotNetStreamReference`. To find examples where I have already done this I recommend that you check out [Blazor.Streams](https://github.com/KristofferStrube/Blazor.Streams) and specifically take a look at [Stream copying to a WritableStream](https://kristofferstrube.github.io/Blazor.Streams/WritableStreamCopyTo) and its associated source code [WritableStreamCopyTo.razor](https://github.com/KristofferStrube/Blazor.Streams/blob/main/samples/KristofferStrube.Blazor.Streams.WasmExample/Pages/WritableStreamCopyTo.razor) which can be adjusted to writing to a `FileSystemWritableFileStream` from the `Blazor.FileSystem` library:
```csharp
 // Some FileSystem stream you have gotten from a FileSystemFileHandle.
FileSystemWritableFileStream fsWritableStream;

// Some Stream of data.
Stream data = await HttpClient.GetStreamAsync("images/mountain.jpg");

// Stream copying.
await data.CopyToAsync(fsWritableStream);
```

Thanks for your interest in the library. 😁
